services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/HankeServiceITests.kt

```diff
@@ -1120,10 +1120,10 @@ class HankeServiceITests(
                     .flatMap { it.contacts }
                     .find { it.orderer }
             assertThat(orderer).isNotNull().all {
-                prop(Contact::firstName).isEqualTo("Teppo")
-                prop(Contact::lastName).isEqualTo("Testihenkilö")
+                prop(Contact::firstName).isEqualTo(ApplicationFactory.TEPPO)
```

Comment:
Could be imported as import fi.hel.haitaton.hanke.factory.ApplicationFactory.Companion.TEPPO as has been done elsewhere.

Reply:
I copied the style from the surrounding code.